packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -837,8 +837,11 @@ describeNoCompat("stashed ops", (getTestObjectProvider) => {
 		await waitForContainerConnection(container2);
 
 		// get new datastore from first container
-		const dataStore2 = await requestFluidObject<ITestFluidObject>(container1, id);
-		const map2 = await requestFluidObject<SharedMap>(dataStore2.runtime, newMapId);
+		const entryPoint = (await container1.getEntryPoint()) as ITestFluidObject;
+		const containerRuntime = entryPoint.context.containerRuntime as ContainerRuntime;
+
+		const response = await containerRuntime.resolveHandle({ url: `/${id}/${newMapId}` });
```

Comment:
I don't think this should be using resolveHandle. It's supposed to be used only for back-compat. You could get the entryPoint (defaultDataStore) from container2 and get the dataStore2's handle via `defaultDataStore.root.get("someDataStore")`. Maybe put "someDataStore" in a variable outside the pendingOps block.

Reply:
Wouldn't getting `defaultDataStore` from `container2` change what this test is doing? Also, if I use `container1` for getting the entryPoint I need to move the `ensureSynchronized()` call above reaching into `defaultDataStore.root` as the handle won't be there.

Reply:
Maybe you are right, I am not sure. I would defer to the test owner to find a way to implement this without resolveHandle unless they explicitly want to use it for some reason. Maybe tag them here or create a follow up task.

Reply:
[AB#6340](https://dev.azure.com/fluidframework/internal/_workitems/edit/6340)